content/departments/tech-ops/tools/computer-setup.md

```diff
@@ -1,28 +1,27 @@
-## Computer requirements
+# Computer requirements
 
 - All full-time teammates must have a Sourcegraph computer
 - All teammates must work from their Sourcegraph computer
-- All teammates must use their Sourcegraph computer for accessing any company data
-- All teammates should adhear to our related usage and access policies:
-  - [Device useage and privacy policy](../process/team_device_usage_privacy.md)
+- All teammates must only use their Sourcegraph computer for accessing company data
```

Comment:
Just adding a link here to ensure it is clear there is a specific definition of this.

```suggestion
- All teammates must only use their Sourcegraph computer for accessing [company data](#company-data)
```